src/components/DataFetchingInReact/DataFetchingInReact.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import "./DataFetchingInReact.css";
+
+const DataFetchingInReact = () => {
+  const [users, setUsers] = useState(null);
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/users")
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) => {
+        setUsers(data);
+      });
+  }, []);
+
+  return (
+    <div>
+      <h1 className="headline">Data Fetching in React</h1>
+      <div className="demo-block">
+        {users &&
+          users.map((user) => (
```

Comment:
And in here, I think we should also check if the users' length is greater than one.